x-pack/plugin/core/src/test/java/org/elasticsearch/test/SecuritySettingsSourceField.java

```diff
@@ -10,6 +10,7 @@
 public final class SecuritySettingsSourceField {
     public static final SecureString TEST_PASSWORD_SECURE_STRING = new SecureString("x-pack-test-password".toCharArray());
     public static final String TEST_PASSWORD = "x-pack-test-password";
+    public static final String TEST_INVALID_PASSWORD = "invalid-test-password";
```

Comment:
I think you can use a randomAlphaNumericOfLength , or a permutation of `TEST_PASSWORD` plus some random string for whenever we need this, other than defining it here. That's just personal preference, no strong views, feel free to keep as is if you prefer it